man/systemd.netdev.xml

```diff
@@ -185,6 +185,9 @@
 
           <row><entry><varname>bareudp</varname></entry>
           <entry> Bare UDP tunnels provide a generic L3 encapsulation support for tunnelling different L3 protocols like MPLS, IP etc. inside of an UDP tunnel.</entry></row>
+
+          <row><entry><varname>batadv</varname></entry>
+          <entry> Batman advanced is a new approach to wireless networking which does no longer operate on the IP basis. Unlike the batman daemon, which exchanges information using UDP packets and sets routing tables, batman-advanced operates on ISO/OSI Layer 2 only and uses and routes (or better: bridges) Ethernet Frames. It emulates a virtual network switch of all nodes participating.</entry></row>
```

Comment:
It'd be nice to add some hyperlinks to "Batman advanced" and "Batman". Most people might not be familiar.